cmd/influxd/main.go

```diff
@@ -28,23 +29,25 @@ func main() {
 
 	influxdb.SetBuildInfo(version, commit, date)
 
-	rootCmd := launcher.NewInfluxdCommand(context.Background(),
-		// FIXME
-		//generate.Command,
-		//restore.Command,
```

Comment:
(As far as I can tell) this was replaced by the backup/restore commands in the `influx` CLI, so it's fine to remove the FIXME without a tracking issue